resources/analysisTools/copyNumberEstimationWorkflow/phasing.sh

```diff
@@ -20,12 +20,12 @@ then
 
     CHR_NR="$CHR_PREFIX${CHR_NAME:?CHR_NAME is not set}"
 
-    $SAMTOOLS_BINARY mpileup "$CNV_MPILEUP_OPTS" -u \
+    $BCFTOOLS_BINARY mpileup $CNV_MPILEUP_OPTS -O u \
```

Comment:
why did you remove the quotes here and in line 28?

Reply:
The translated shell script still contained the quotes, this created the issue during `mpileup` calls.

Reply:
which value of CNV_MPILEUP_OPTS  caused issues for mpileup using quotes and no issue without quotes?

Reply:
Thank you for pointing me to
https://github.com/DKFZ-ODCF/ACEseqWorkflow/pull/18#discussion_r576230951
Now I see what you meant.